src/NodeManipulator/IfManipulator.php

```diff
@@ -336,6 +325,33 @@ public function isIfCondUsingAssignNotIdenticalVariable(If_ $if, Node $node): bo
         return ! $this->betterStandardPrinter->areNodesEqual($this->getIfCondVar($if), $node->var);
     }
 
+    public function createIfNegation(Expr $expr, Stmt $stmt): If_
+    {
+        $expr = $expr instanceof BooleanNot
+            ? $expr->expr
+            : $this->createNegationExpr($expr);
+
+        return new If_(
+            $expr,
+            [
+                'stmts' => [$stmt],
+            ]
+        );
+    }
+
+    private function createNegationExpr(Expr $expr): Expr
+    {
+        if ($expr instanceof Identical) {
+            return new NotIdentical($expr->left, $expr->right);
+        }
+
+        if ($expr instanceof NotIdentical) {
+            return new Identical($expr->left, $expr->right);
+        }
+
+        return new BooleanNot($expr);
+    }
```

Comment:
Might be useful `Rector\EarlyReturn\NodeTransformer\ConditionInverter`

Reply:
implemented.